e2e/timeout-file-not-cached/src/BetterStandardPrinter.php

```diff
@@ -0,0 +1,625 @@
+<?php
+
+declare(strict_types=1);
+
+
+use Nette\Utils\Strings;
+use PhpParser\Comment;
+use PhpParser\Node;
+use PhpParser\Node\Expr;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ArrowFunction;
+use PhpParser\Node\Expr\Assign;
+use PhpParser\Node\Expr\Closure;
+use PhpParser\Node\Expr\Ternary;
+use PhpParser\Node\Expr\Yield_;
+use PhpParser\Node\Name;
+use PhpParser\Node\Name\FullyQualified;
+use PhpParser\Node\Param;
+use PhpParser\Node\Scalar\DNumber;
+use PhpParser\Node\Scalar\EncapsedStringPart;
+use PhpParser\Node\Scalar\LNumber;
+use PhpParser\Node\Scalar\String_;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\ClassMethod;
+use PhpParser\Node\Stmt\Declare_;
+use PhpParser\Node\Stmt\InlineHTML;
+use PhpParser\Node\Stmt\Nop;
+use PhpParser\Node\Stmt\Use_;
+use PhpParser\PrettyPrinter\Standard;
+use Rector\Comments\NodeDocBlock\DocBlockUpdater;
+use Rector\Core\Configuration\RectorConfigProvider;
+use Rector\Core\Contract\PhpParser\NodePrinterInterface;
+use Rector\Core\NodeDecorator\MixPhpHtmlDecorator;
+use Rector\Core\PhpParser\Node\CustomNode\FileWithoutNamespace;
+use Rector\Core\Provider\CurrentFileProvider;
+use Rector\Core\Util\StringUtils;
+use Rector\Core\ValueObject\Application\File;
+use Rector\Core\ValueObject\Reporting\FileDiff;
+use Rector\NodeTypeResolver\Node\AttributeKey;
+
+/**
+ * @see \Rector\Core\Tests\PhpParser\Printer\BetterStandardPrinterTest
+ *
+ * @property array<string, array{string, bool, string, null}> $insertionMap
+ */
+final class BetterStandardPrinter extends Standard implements NodePrinterInterface
```

Comment:
Please rename so it not make collision when using IDE with the actual inside `src` one, eg:

```suggestion
final class SomeFixturePrinter extends Standard implements NodePrinterInterface
```